Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -150,6 +150,15 @@ extension BrowserViewController {
                 }
         }
     }
+    
+    // Note: at the moment the toggle value does not matter, the same code is run regardless of its state.
+    @objc func rewardsSettingToggled() {
```

Comment:
I can do 1 selector.
We do not care about value because it just resets the NTP notification, and what notification is shown depends on few conditions, rewards state, ads state, image type. Does not need to be shown immediately.